data_pipelines_cli/cli_commands/deploy.py

```diff
@@ -38,11 +38,7 @@ class DeployCommand:
     e.g. path to a token, username, password, etc."""
     env: str
     bi_git_key_path: str
-    """Path to JSON file containing key for GCP service account
-    used to communicate with IAP-secured applications"""
-    gcp_sa_key_path: Optional[str]
-    """Client ID of Airbyte IAP-secured instance"""
-    airbyte_iap_client_id: Optional[str]
+    auth_token: Optional[str]
```

Comment:
You could add few words about this var for example 
`"""Authorization OIDC ID token for a service account to communication with Airbyte instance"""`